.travis.yml

```diff
@@ -3,21 +3,22 @@ python:
   - "3.6"

```

Comment:
I think we are using Python 3.8 in our environment?

Reply:
Good Catch
Will change it to 3.8